LayoutTests/fast/events/touch/ios/touch-event-regions-layer-tree/dblclick-window.html

```
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)